src/applications/caregivers/tests/components/FormReview/AddressWithAutofillReviewField.unit.spec.js

```diff
@@ -30,9 +18,26 @@ describe('CG <AddressWithAutofillReviewField>', () => {
     ),
   });
 
+  const defaultProps = {
+    canAutofillAddress: false,
+    formData: {
+      street: '1350 I St. NW',
+      street2: 'Suite 550',
+      city: 'Washington',
+      state: 'DC',
+      postalCode: '20005',
+      'view:autofill': false,
+    },
+    inputLabel: 'Primary Family Caregiver\u2019s',
+  };
+
+  const subject = (props = defaultProps) => {
```

Comment:
I wrapped our `render` and `getSelectors` calls into a subject and set the `defaultProps` on it with the ability to pass in custom props.